travel/Html/travel_platform.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
Well, if you're trying to implement logic with base template, you don't need to write full HTML in separate file. You just need to extend base template with some specific information.